dashboard/app/views/videos/embed.html.haml

```diff
@@ -3,7 +3,7 @@
 !!! 5
 - video_info = local_assigns[:video_info]
 %head
-  = stylesheet_link_tag    '/assets/video-js/video-js', media: 'all'
+  = stylesheet_link_tag '/blockly/video-js/video-js', media: 'all'
```

Comment:
Can we track down what asset-build-system change originally prompted the need for this path change? I assume this must have been working correctly at some point, and it would be good to verify whether we might have missed anything else that might also need to be updated as a result of the original change.

Reply:
Current suspect is this line: https://github.com/code-dot-org/code-dot-org/commit/04f69b04f319067dc7b3c9dac8fd327ff4acd1a0#diff-239b7a0365c3ae953719e57e0ded25d2L37

Reply:
That older technique did have the advantage of not introducing a new standalone CSS file to be retrieved.

Reply:
So we think this has been broken since July 2016? Seems like we should probably add a test case to catch this in any case.

Reply:
good find! (for x-ref purposes, the PR is #9340, and it seems that this exact issue was already tracked down and fixed in #12829, but not for all video-player instances.)

Is the TODO in the comment linked above still relevant, or could it be updated to reflect the changes/learnings in this PR?

Reply:
Since #12829 got it appearing in /assets/ I think it got it working for localhost but not for other environments, but maybe there's more subtlety here.

As for the comment, it seems we could add ```@import 'video.js/dist/video-js/video-js';``` to https://github.com/code-dot-org/code-dot-org/blob/staging/apps/style/code-studio/code-studio.scss, which seems better than keeping it standalone.

However, then it has trouble loading the web font, which doesn't exist: 
http://localhost-studio.code.org:3000/assets/css/font/vjs.woff

Versus where it currently loads successfully: 
http://localhost-studio.code.org:3000/blockly/video-js/font/vjs.woff

It seems we might be able to get the font outputting to `/assets/css/font` and then have the CSS refer to this file, which is presumably fingerprinted, but I don't know the system very well.  Thoughts?